server/src/infra/entities/asset.entity.ts

```diff
@@ -148,6 +148,16 @@ export class AssetEntity {
 
   @OneToMany(() => AssetFaceEntity, (assetFace) => assetFace.asset)
   faces!: AssetFaceEntity[];
+
+  @Column({ type: 'varchar', nullable: true })
```

Comment:
This type isn't actually accurate, since it is a uuid from the relation. Just remove `type: 'varchar'` entirely.

Reply:
Updated.